src/modules/job-manager/jobtap.c

```diff
@@ -32,6 +32,7 @@
 
 #include "annotate.h"
 #include "prioritize.h"
+#include "conf.h"
```

Comment:
typo (double "now") in commit message for 4c3545e7d91180293c733250900f74440add2292

> but now job-manager now supports